Sources/SwiftCurrent/Models/Workflow.swift

```diff
@@ -285,7 +285,7 @@ extension Workflow {
 
 extension Workflow where F.WorkflowOutput == Never {
     /**
-     Adds an item to the workflow; enforces the `FlowRepresentable.WorkflowOutput` of the previous item matches the `FlowRepresentable.WorkflowInput` of this item.
+     Adds an item to the workflow, and enforces the `FlowRepresentable.WorkflowOutput` of the previous item matches the `FlowRepresentable.WorkflowInput` of this item.
```

Comment:
This goes against the Swift API Style Guide

Reply:
Reverted all instances of this change